conf/modules.config

```diff
@@ -158,6 +158,13 @@ process {
             pattern: "*"
         ]
     }
+    withName: 'HMMER' {
+        publishDir = [
+            path: { "${params.outdir}/hmmsearch" },
+            mode: 'copy',
+            pattern: "*"
+        ]
+    }
```

Comment:
Is this necessary, or just something left? The one below `HMMER_HMMSEARCH` looks like a better match.

Reply:
Yes, we don't need it. I'll remove it